packages/pwa-buildpack/lib/Utilities/__tests__/__snapshots__/createDotEnvFile.spec.js.snap

```diff
@@ -54,6 +54,10 @@ MAGENTO_BACKEND_EDITION=CE
 #   - Default when not set: 
 STORE_VIEW_CODE=
 #
+#   Specify whether or not the store code URLs should be appended to the URLs of the store.
+#   - Default when not set: false
+USE_STORE_CODE_IN_URL=false
```

Comment:
Needs added in docs as well

Reply:
@jcalcaben tells me the docs for the .env file is automatically generated.